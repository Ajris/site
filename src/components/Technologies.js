import React, {Component} from 'react';

export default class Technologies extends Component {
    render() {
        return (
            <div className={'mx-4 my-3'}>
                <p className={'topic text-center'}>TECHNOLOGIES INFORMATION</p>
                <p className={'topic text-center'}>TECHNOLOGIES INFORMATION</p>
                <p className={'topic text-center'}>TECHNOLOGIES INFORMATION</p>
                <p className={'topic text-center'}>TECHNOLOGIES INFORMATION</p>
                <p className={'topic text-center'}>TECHNOLOGIES INFORMATION</p>
                <p className={'topic text-center'}>TECHNOLOGIES INFORMATION</p>
            </div>
        )
    }
}
