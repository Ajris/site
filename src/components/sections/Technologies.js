import React from 'react';
import Card from "./Card";

export default class Technologies extends React.Component {
    render() {
        return (
            <div>
                <div className={'mx-4 my-3'}>
                    <p className={'topic text-center'}>Technologies</p>
                </div>
                <div className="card-lines">
                    <div className="card-deck">
                        <Card title="HOME 1" text="TEXT 1" footer="footer 1"/>
                        <Card title="HOME 1" text="TEXT 1" footer="footer 1"/>
                        <Card title="HOME 1" text="TEXT 1" footer="footer 1"/>
                    </div>
                    <div className="card-deck">
                        <Card title="HOME 1" text="TEXT 1" footer="footer 1"/>
                        <Card title="HOME 1" text="TEXT 1" footer="footer 1"/>
                        <Card title="HOME 1" text="TEXT 1" footer="footer 1"/>
                    </div>
                </div>
            </div>
        )
    }
}
