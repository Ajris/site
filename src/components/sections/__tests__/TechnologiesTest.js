import React from "react";
import { create } from "react-test-renderer";
import Technologies from "../Technologies";

// describe("Technologies component", () => {
//     test("it matches the snapshot", () => {
//         const component = create(<Technologies/>);
//         expect(component.toJSON()).toMatchSnapshot();
//     });
// });
