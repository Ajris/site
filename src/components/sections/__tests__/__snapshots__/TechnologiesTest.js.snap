// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Technologies component it matches the snapshot 1`] = `
<div>
  <div
    className="mx-4 my-3"
  >
    <p
      className="topic text-center"
    >
      Technologies
    </p>
  </div>
  <div
    className="container"
  >
    <div
      className="justify-content-md-center row"
    />
  </div>
</div>
`;
